services/ui-src/src/components/reports/ReportPageFooter.tsx

```diff
@@ -15,14 +15,25 @@ export const ReportPageFooter = ({
   ...props
 }: Props) => {
   const navigate = useNavigate();
-  const formIsDisabled = false;
-  const report = useStore().report;
+  const { report, editable, selectedEntity } = useStore();
   const { previousRoute, nextRoute } = useFindRoute(
     report?.formTemplate.flatRoutes,
     report?.formTemplate?.basePath
   );
   const hidePrevious = previousRoute === "/wp" || previousRoute === "/sar";
 
+  /*
+   * By default, the Continue button submits the form AND navigates.
+   * If the form is disabled, the button will navigate only.
+   */
+  const reportWithSubmittedStatus = report?.status === ReportStatus.SUBMITTED;
+  const { userIsAdmin, userIsEndUser } = useStore().user ?? {};
+  const formIsDisabled =
```

Comment:
While we are in here, is it worth trying to write a test for this? I.e., ensure that the continue button calls the useNavigate hook if the form is disabled, instead of a submit? 

Not a blocker, because sometimes tests are more complex than it's worth, but wondering how you feel about covering our behinds for this scenario. 😆

Reply:
Done! Thanks for the nudge :)